tests/Fixtures/TestBundle/Entity/DummyTableInheritanceNotApiResourceChild.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class DummyTableInheritanceNotApiResourceChild extends DummyTableInheritance
```

Comment:
`NonApiResourceTableInheritedDummyChild` would be better? @teohhanhui do you have a name in mind? :p

Reply:
Well, starting by "DummyTableInheritance" seems to have been a convention for similar objects.